assets/cms/scss/panels/_shared.scss

```diff
@@ -440,6 +452,64 @@ div.ccm-panel-detail-form-actions {
   padding: 15px 20px 15px 340px;
   position: fixed;
   width: 100%;
+
+  &.side-panel {
+    padding: 10px;
+  }
+}
+
+/**
+ * Page Design Side Panel
+ */
+#ccm-panel-page-design {
```

Comment:
We should be putting these rules in a separate scss file. For example we have versions SCSS in `panels/pages/_versions.scss`, let's put this in `panels/pages/_design.scss`